plugins/jobs/plugin.zsh

```diff
@@ -0,0 +1,14 @@
+geometry_prompt_jobs_setup() {
+  return true
+}
+
+geometry_prompt_jobs_check() {
+  return true
+}
```

Comment:
this is unnecessary - if the check function is not defined, it defaults to return true. If you like it for consistency or as an example its fine to keep.